token-metadata/program/src/instruction/mod.rs

```diff
@@ -195,6 +201,7 @@ pub enum MetadataInstruction {
     #[account(11, name="token_program", desc="Token program")]
     #[account(12, name="system_program", desc="System program")]
     #[account(13, optional, name="rent", desc="Rent info")]
+    #[account(14, name="sysvar_instructions", desc="Instructions sysvar account")]
```

Comment:
Adding a comment here to double-check whether the `metadata` needs to be mutable or not. Probably not since the `new_metadata` is mutable.

Reply:
As long as one of the metadata accounts is mutable, we should be covered.